src/cpp/session/modules/SessionConnectionsInstaller.R

```diff
@@ -611,5 +612,9 @@
 })
 
 .rs.addFunction("connectionOdbcRStudioDriver", function() {
-   " with RStudio Driver"
+   " with Posit Driver"
+})
+
+.rs.addFunction("connectionStripRStudioDriver", function(connection) {
```

Comment:
I'm not a huge fan of this name, but I wanted to keep it similar to `connectionOdbcRStudioDriver`